src/Avalonia.Controls/Automation/Peers/ScrollBarAutomationPeer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Avalonia.Controls.Primitives;
+
+namespace Avalonia.Automation.Peers
+{
+    public class ScrollBarAutomationPeer : RangeBaseAutomationPeer
+    {
+        public ScrollBarAutomationPeer(ScrollBar owner) : base(owner)
+        {
+        }
+
+        override protected string GetClassNameCore()
+        {
+            return "ScrollBar";
```

Comment:
Should this use `nameof()` ? Magic strings are generally frowned upon and I've noticed this with all the other automation peers as well.